examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -599,6 +615,27 @@ def remove_special_characters(batch):
     # derive max & min input length for sample rate & max duration
     max_input_length = data_args.max_duration_in_seconds * feature_extractor.sampling_rate
     min_input_length = data_args.min_duration_in_seconds * feature_extractor.sampling_rate
+
+    if data_args.max_eval_duration_in_seconds is None:
+        logger.warning(
+            "`max_eval_duration_in_seconds` is not provided. Filtering audio samples in the eval set longer than"
```

Comment:
```suggestion
            f"`max_eval_duration_in_seconds` is not provided and `max_duration_in_seconds` is set to {max_duration_in_seconds}. Generally, it is not recommended to filter evaluation data, so we strongly recommend setting `max_eval_duration_in_seconds` either explicictely or setting it to `False`. For backwards compatibility,  audio samples in the eval set longer than `max_duration_in_seconds`: {max_duration_in_seconds} are filtered."
```

Reply:
I think we should offer the user here an option to easily disable filtering

Reply:
See comment above about allowing `max_eval_duration_in_seconds` to be a boolean set to False